_posts/2018-08-31-grpc-packaging-releases-nightlies.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: gRPC Packaging: Release Process and Nightly Builds
+published: true
+permalink: blog/grpc-packaging-releases-nightlies
+author: Mehrdad Afshari
+author-link: https://mehrdad.afshari.me
+company: Google
+company-link: https://www.google.com
+---
+
+# gRPC Packaging: Release Process and Nightly Builds #
+
+gRPC is [developed in the open on GitHub](https://github.com/grpc) and the
+`master` branch undergoes continuous testing to try to ensure the `HEAD` of
+`master` is buildable and passes the tests.  In this post, we discuss the
+release process of gRPC as an open source project to clarify it for our users,
+and to serve as an example for other open source projects to consider adopting.
+This article is written mostly from the perspective of the gRPC Core (C-based)
+languages.  The release processes of gRPC Java and Go, although similar in
+spirit, may be slightly different in technicalities.
+
+## Official Releases ##
+
+gRPC follows semantic versioning and we aim to ship a new minor version every
+six weeks.  While `master` is quite good and heavily tested, and we could
+almost always simply consider tagging its `HEAD` as a minor release whenever we
+like to ship one, a more rigorous quality control process is justified for
+supported gRPC releases, as it is a critical piece of infrastructure for many
+users.  As a consequence, we schedule a feature freeze every six weeks.  [gRPC
+Release Schedule]() highlights the actual dates.
+
+### Feature Freeze: Release Branch Cut ###
+
+The feature window for each minor release closes when a release branch is cut
+from `master`.  The release manager of that particular release takes a commit
+close to the `HEAD`, often the `HEAD` itself, where the tests are green, and
+creates a new release branch at that point.  Starting at the branch cut point,
+additional feature development continues on the `master` branch without
+affecting the release branch and changes to the release branch are kept
+minimal, *i.e.* mostly bug fixes.  The version number on the `master` branch is
+promptly bumped to reflect that new features will land starting in the followup
+release.  The `master` branch for gRPC Core repository always carries a `-dev`
+suffix to indicate it is under active development.
```

Comment:
Do you want to state here that the branch cut date is a hard rolling deadline of every 6 weeks?